features/fields_extensie.feature

```diff
@@ -1,126 +1,570 @@
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En heeft de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                                           |
+      | fields   | fields        | Parameter bevat een niet bestaande propertynaam. |
+
+    Scenario: Fields parameter met attribuutnaam met onjuiste case
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                   |
+      | type                | RaadpleegMetBurgerservicenummer          |
+      | burgerservicenummer | 999994086                                |
+      | fields              | BurgerServiceNummer,geslachtsaanduiding  |
+      Dan heeft de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En heeft de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                                                |
+      | fields   | fields        | Parameter bevat een porpertynaam met onjuiste casing. |
+
+  Rule: properties die met dezelfde naam in meerdere groepen voorkomen moeten met een .(dot)-notatie worden opgenomen in fields
+    Scenario: opvragen property waarvan de naam in meerdere groepen voorkomt met .(dot)-notatie in de fields
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
```

Comment:
geslachtsaanduiding is een tabelwaarde type. Hiermee is niet duidelijk welke property van tabelwaarde moet worden gevuld. Gebruik of 'geslachtsaanduiding (04.10)' (automation code doet de vertaling) of geslachtsaanduiding.code